scripts/kind-e2e/e2e.sh

```diff
@@ -204,6 +199,39 @@ function kind_import_images() {
     done
 }
 
+function create_subm_vars() {
```

Comment:
If we're going to move this into the main e2e script, we should likely also use these vars in the Helm deploys (`setup_cluster2_gateway`, `setup_cluster3_gateway`).

Reply:
That's not the intention for the change, the intention is to get the helm e2e back working